src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/Link.js

```diff
@@ -69,6 +72,16 @@ class Link extends Component<Props> {
             return;
         }
 
+        if (value.provider === 'external') {
+            const {href} = value;
+
+            if (href) {
+                this.titleParts = [href];
+
+                return;
+            }
+        }
+
```

Comment:
Currently the `link` content type doesn't show a value, if the `external` provider is used. I know, that the `Link` container should not know about `external`, but I'm not sure how we should fix that. Checking if `linkTypeRegistry.getOptions(value.provider)` returns `undefined` is a bit hacky and might break in the future.

Reply:
i think i would always use `value.href` as title if we dont have any `options` or `options.displayProperties`. then we dont need to check the type and i think its better to display anything than display nothing 🙂

Reply:
Sounds good for me fallbacking always to `href` in that case.